### PR TITLE
Fixes page aspect ratio to 6/19

### PR DESCRIPTION
**Pull request by prateekbh**
This pr fixes page aspect ratio to 6/19 as listed in #11848

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

File: extensions/amp-story/0.1/amp-story.js [hunk 1]
```diff
@@ -609,15 +609,19 @@ export class AmpStory extends AMP.BaseElement {
     const platform = Services.platformFor(this.win);
     if (platform.isSafari() || platform.isIos()) {
       this.mutateElement(() => {
-        setStyle(this.element, 'display', 'none');
+        setImportantStyles(this.element, {
```
Comment (choumx) [inline extensions/amp-story/0.1/amp-story.js]: I thought we only want to set initial amp-story styles as important.

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: ah sorry, my bad. Didn't see that the one in file is also not important

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: Done

**Comment by prateekbh** [comment]
Any idea why CLA is stuck?

**Comment by prateekbh** [comment]
request for merge

**Pull request merged**